menas/src/main/scala/za/co/absa/enceladus/menas/repositories/DatasetMongoRepository.scala

```diff
@@ -65,40 +65,35 @@ class DatasetMongoRepository @Autowired()(mongoDb: MongoDatabase)
   }
 
   override def create(item: Dataset, username: String): Future[Completed] = {
-    super.create(handleMappingRuleWrite(item), username)
+    super.create(handleMappingRuleWrite(item), username)//
```

Comment:
```suggestion
    super.create(handleMappingRuleWrite(item), username)
```

Probably forgot to remove with the temp comment.